tests/managed-resource-edit-selection-test.js

```diff
@@ -0,0 +1,171 @@
+import { beforeEach, afterEach, describe, it } from '@bigtest/mocha';
```

Comment:
Let's rename this test. `resource-edit-managed-title-in-custom-package-test`? Long, I know, but we're not testing a managed resource here. Alternate suggestions encouraged.

Reply:
Yeah that title is very accurate.